Resolves #85

palemieux opened the pull request:
Resolves #85

nigelmegitt commented:
Looks fine to me. One thing I just noticed is a bit strange is that for the image profile a _presented image_ is a `div` with a `smpte:backgroundImage` attribute, and that at most one _presented image_ shall be present per presented region. Yet there's no additional provision against `#content` stating that at most one `div` element shall be temporally active within any region at any moment in time. Worth adding?

palemieux commented:
> Yet there's no additional provision against #content stating that at most one div element shall be temporally active within any region at any moment in time. Worth adding?

Added reference to "presented image" constraints at \#content.

Added missing \#content to text profile.

nigelmegitt commented:
Looks good to me. I did spot a separate problem with §8.2.1 but will file separately.
